### ISSUE TITLE
https://huggingface.co/facebook/detr-resnet-101-panoptic model has 250 classes ?

### ISSUE THREAD

**Issue by zhangbo2008** [huggingface/transformers#12808]
coco data has 80 classes. 
why the model has 250 classes

id2label: {
    "0": "LABEL_0",
    "1": "LABEL_1",
    "2": "LABEL_2",
    "3": "LABEL_3",
    "4": "LABEL_4",
    "5": "LABEL_5",
    "6": "LABEL_6",
    "7": "LABEL_7",
    "8": "LABEL_8",
    "9": "LABEL_9",
    "10": "LABEL_10",
    "11": "LABEL_11",
    "12": "LABEL_12",
    "13": "LABEL_13",
    "14": "LABEL_14",
    "15": "LABEL_15",
    "16": "LABEL_16",
    "17": "LABEL_17",
    "18": "LABEL_18",
    "19": "LABEL_19",
    "20": "LABEL_20",
    "21": "LABEL_21",
    "22": "LABEL_22",
    "23": "LABEL_23",
    "24": "LABEL_24",
    "25": "LABEL_25",
    "26": "LABEL_26",
    "27": "LABEL_27",
    "28": "LABEL_28",
    "29": "LABEL_29",
    "30": "LABEL_30",
    "31": "LABEL_31",
    "32": "LABEL_32",
    "33": "LABEL_33",
    "34": "LABEL_34",
    "35": "LABEL_35",
    "36": "LABEL_36",
    "37": "LABEL_37",
    "38": "LABEL_38",
    "39": "LABEL_39",
    "40": "LABEL_40",
    "41": "LABEL_41",
    "42": "LABEL_42",
    "43": "LABEL_43",
    "44": "LABEL_44",
    "45": "LABEL_45",
    "46": "LABEL_46",
    "47": "LABEL_47",
    "48": "LABEL_48",
    "49": "LABEL_49",
    "50": "LABEL_50",
    "51": "LABEL_51",
    "52": "LABEL_52",
    "53": "LABEL_53",
    "54": "LABEL_54",
    "55": "LABEL_55",
    "56": "LABEL_56",
    "57": "LABEL_57",
    "58": "LABEL_58",
    "59": "LABEL_59",
    "60": "LABEL_60",
    "61": "LABEL_61",
    "62": "LABEL_62",
    "63": "LABEL_63",
    "64": "LABEL_64",
    "65": "LABEL_65",
    "66": "LABEL_66",
    "67": "LABEL_67",
    "68": "LABEL_68",
    "69": "LABEL_69",
    "70": "LABEL_70",
    "71": "LABEL_71",
    "72": "LABEL_72",
    "73": "LABEL_73",
    "74": "LABEL_74",
    "75": "LABEL_75",
    "76": "LABEL_76",
    "77": "LABEL_77",
    "78": "LABEL_78",
    "79": "LABEL_79",
    "80": "LABEL_80",
    "81": "LABEL_81",
    "82": "LABEL_82",
    "83": "LABEL_83",
    "84": "LABEL_84",
    "85": "LABEL_85",
    "86": "LABEL_86",
    "87": "LABEL_87",
    "88": "LABEL_88",
    "89": "LABEL_89",
    "90": "LABEL_90",
    "91": "LABEL_91",
    "92": "LABEL_92",
    "93": "LABEL_93",
    "94": "LABEL_94",
    "95": "LABEL_95",
    "96": "LABEL_96",
    "97": "LABEL_97",
    "98": "LABEL_98",
    "99": "LABEL_99",
    "100": "LABEL_100",
    "101": "LABEL_101",
    "102": "LABEL_102",
    "103": "LABEL_103",
    "104": "LABEL_104",
    "105": "LABEL_105",
    "106": "LABEL_106",
    "107": "LABEL_107",
    "108": "LABEL_108",
    "109": "LABEL_109",
    "110": "LABEL_110",
    "111": "LABEL_111",
    "112": "LABEL_112",
    "113": "LABEL_113",
    "114": "LABEL_114",
    "115": "LABEL_115",
    "116": "LABEL_116",
    "117": "LABEL_117",
    "118": "LABEL_118",
    "119": "LABEL_119",
    "120": "LABEL_120",
    "121": "LABEL_121",
    "122": "LABEL_122",
    "123": "LABEL_123",
    "124": "LABEL_124",
    "125": "LABEL_125",
    "126": "LABEL_126",
    "127": "LABEL_127",
    "128": "LABEL_128",
    "129": "LABEL_129",
    "130": "LABEL_130",
    "131": "LABEL_131",
    "132": "LABEL_132",
    "133": "LABEL_133",
    "134": "LABEL_134",
    "135": "LABEL_135",
    "136": "LABEL_136",
    "137": "LABEL_137",
    "138": "LABEL_138",
    "139": "LABEL_139",
    "140": "LABEL_140",
    "141": "LABEL_141",
    "142": "LABEL_142",
    "143": "LABEL_143",
    "144": "LABEL_144",
    "145": "LABEL_145",
    "146": "LABEL_146",
    "147": "LABEL_147",
    "148": "LABEL_148",
    "149": "LABEL_149",
    "150": "LABEL_150",
    "151": "LABEL_151",
    "152": "LABEL_152",
    "153": "LABEL_153",
    "154": "LABEL_154",
    "155": "LABEL_155",
    "156": "LABEL_156",
    "157": "LABEL_157",
    "158": "LABEL_158",
    "159": "LABEL_159",
    "160": "LABEL_160",
    "161": "LABEL_161",
    "162": "LABEL_162",
    "163": "LABEL_163",
    "164": "LABEL_164",
    "165": "LABEL_165",
    "166": "LABEL_166",
    "167": "LABEL_167",
    "168": "LABEL_168",
    "169": "LABEL_169",
    "170": "LABEL_170",
    "171": "LABEL_171",
    "172": "LABEL_172",
    "173": "LABEL_173",
    "174": "LABEL_174",
    "175": "LABEL_175",
    "176": "LABEL_176",
    "177": "LABEL_177",
    "178": "LABEL_178",
    "179": "LABEL_179",
    "180": "LABEL_180",
    "181": "LABEL_181",
    "182": "LABEL_182",
    "183": "LABEL_183",
    "184": "LABEL_184",
    "185": "LABEL_185",
    "186": "LABEL_186",
    "187": "LABEL_187",
    "188": "LABEL_188",
    "189": "LABEL_189",
    "190": "LABEL_190",
    "191": "LABEL_191",
    "192": "LABEL_192",
    "193": "LABEL_193",
    "194": "LABEL_194",
    "195": "LABEL_195",
    "196": "LABEL_196",
    "197": "LABEL_197",
    "198": "LABEL_198",
    "199": "LABEL_199",
    "200": "LABEL_200",
    "201": "LABEL_201",
    "202": "LABEL_202",
    "203": "LABEL_203",
    "204": "LABEL_204",
    "205": "LABEL_205",
    "206": "LABEL_206",
    "207": "LABEL_207",
    "208": "LABEL_208",
    "209": "LABEL_209",
    "210": "LABEL_210",
    "211": "LABEL_211",
    "212": "LABEL_212",
    "213": "LABEL_213",
    "214": "LABEL_214",
    "215": "LABEL_215",
    "216": "LABEL_216",
    "217": "LABEL_217",
    "218": "LABEL_218",
    "219": "LABEL_219",
    "220": "LABEL_220",
    "221": "LABEL_221",
    "222": "LABEL_222",
    "223": "LABEL_223",
    "224": "LABEL_224",
    "225": "LABEL_225",
    "226": "LABEL_226",
    "227": "LABEL_227",
    "228": "LABEL_228",
    "229": "LABEL_229",
    "230": "LABEL_230",
    "231": "LABEL_231",
    "232": "LABEL_232",
    "233": "LABEL_233",
    "234": "LABEL_234",
    "235": "LABEL_235",
    "236": "LABEL_236",
    "237": "LABEL_237",
    "238": "LABEL_238",
    "239": "LABEL_239",
    "240": "LABEL_240",
    "241": "LABEL_241",
    "242": "LABEL_242",
    "243": "LABEL_243",
    "244": "LABEL_244",
    "245": "LABEL_245",
    "246": "LABEL_246",
    "247": "LABEL_247",
    "248": "LABEL_248",
    "249": "LABEL_249"
  }

and what is the trully name of the 250 classes?

## Environment info

- `transformers` version:
- Platform:
- Python version:
- PyTorch version (GPU?):
- Tensorflow version (GPU?):
- Using GPU in script?:
- Using distributed or parallel set-up in script?:

## Information

Model I am using (Bert, XLNet ...):

The problem arises when using:
* [ ] the official example scripts: (give details below)
* [ ] my own modified scripts: (give details below)

The tasks I am working on is:
* [ ] an official GLUE/SQUaD task: (give the name)
* [ ] my own task or dataset: (give details below)

## To reproduce

1.
2.
3.

**Comment by NielsRogge** [comment]
Yes, the authors just set it to a sufficiently high number to make sure all classes are covered. See https://github.com/facebookresearch/detr/issues/175#issue-672014097

Note that for the panoptic model, one combines the COCO instance classes with stuff classes (background things like trees, streets, sky,...).

**Comment by zhangbo2008** [comment]
thanks a lot bro.
